arches/app/templates/views/components/search/map-filter.htm

```diff
@@ -2,11 +2,11 @@
 {% load i18n %}
 
 {% block tabs %}
-<div class="workbench-card-sidebar-tab" data-bind="click: function() {
-    toggleTab('filter');
+<div tabindex="0" id="toggle-map-filter-panel" class="workbench-card-sidebar-tab" data-bind="onEnterkeyClick, onSpaceClick, click: function() {
+    toggleTab('filter'); toggleAriaExpanded('toggle-map-filter-panel');
 }, css: {
     'active': activeTab() === 'filter'
-}">
+}" aria-label="Click to open map filter" aria-controls="side-panel" aria-expanded="false">
```

Comment:
all `aria-label` values need to be localized by creating an entry in `javascript.htm` and then adding a reference to the `$root.translations` object.  See other parts of this file for an example.

Reply:
Fixed in commit 62e4a05